tests/conftest.py

```
@@ -83,11 +83,22 @@ def gui(qapp, _session_config_root):
+        for _timer_name in (
+            "_conn_test_debounce",
+            "_status_debounce",
+            "_cleanup_timer",
+            "binary_debounce",
+        ):
+            _timer = getattr(g, _timer_name, None)
+            if _timer is not None:
+                try:
+                    _timer.stop()
+                except Exception:
+                    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)